### PR TITLE
Reviews index

### PR DESCRIPTION
**Pull request by jhayes00**
Finished reviews index along with testing.

### CODE REVIEW

File: app/javascript/react/components/GameTile.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
 
+
```
Comment (pmwals09) [inline app/javascript/react/components/GameTile.js:4]: Should we get rid of this extra return?

File: app/javascript/react/components/GamesIndexContainer.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, {useState, useEffect} from 'react'
 import GameTile from './GameTile'
 
+
```
Comment (pmwals09) [inline app/javascript/react/components/GamesIndexContainer.js:4]: Should we get rid of this extra return?

File: app/javascript/react/components/ReviewTile.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+
```
Comment (pmwals09) [inline app/javascript/react/components/ReviewTile.js:3]: Extra return

File: app/javascript/react/components/ReviewTile.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+const ReviewTile = props => {
+  const rating = props.rating 
+  const body = props.body 
+    
+  return(
+    <div>
+      <li>
```
Comment (pmwals09) [inline app/javascript/react/components/ReviewTile.js:10]: I think the ul and li tags are reversed here. Also, do you want the rating and the body to both be list items? If so:
```
<ul>
  <li>{rating} start</li>
  <li>review: {body}</li>
</ul>
```

File: app/javascript/react/components/ReviewTile.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        {rating} stars
+        <ul>
+          review: {body}
```
Comment (pmwals09) [inline app/javascript/react/components/ReviewTile.js:13]: Can we capitalize?

File: app/javascript/react/components/ReviewsIndexContainer.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import ReviewTile from './ReviewTile'
+
+
```
Comment (pmwals09) [inline app/javascript/react/components/ReviewsIndexContainer.js:4]: Extra return

File: config/routes.rb [hunk 7]
```diff
@@ -9,7 +9,9 @@
 
   namespace :api do
     namespace :v1 do
-      resources :games, only: [:index, :show, :create]
```
Comment (pmwals09) [inline config/routes.rb:12]: Don't we want to include create in here so we can add games?

File: app/javascript/react/components/GameShowTile.js [hunk 8]
```diff
@@ -1,10 +1,12 @@
 import React from 'react'
+import ReviewsIndexContainer from './ReviewsIndexContainer'
```
Comment (safang99) [inline app/javascript/react/components/GameShowTile.js:4]: remove one of the duplicate empty lines here

File: app/javascript/react/components/GameShowTile.js [hunk 9]
```diff
@@ -1,10 +1,12 @@
...
 
 
 const GameShowTile = props => {
   const name = props.name
   const playerNum = props.playerNum
   const description = props.description
+  const reviews = props.reviews
```
Comment (safang99) [inline app/javascript/react/components/GameShowTile.js:10]: We can even destructure this further like so,
```
const GameShowTile = (props) => {
  const { name, playerNum, description, reviews } = props
```
which will create the same variables

File: spec/models/review_spec.rb [hunk 10]
```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  it { should have_valid(:rating).when(3)}
+  it { should have_valid(:body).when("i am reviewing this.")}
+
+  it { should_not have_valid(:rating).when(nil, "", 6, -1, "three") }
+
```
Comment (safang99) [inline spec/models/review_spec.rb:8]: remove the empty line 8

**Pull request merged**